src/com/visualization/piechart/piechart.less

```diff
@@ -21,6 +21,7 @@
 
```

Comment:
This might seem like a weird suggestion, but perhaps `.piechart-figure` should be named `.chart-pie` (as there are potentially other types of charts). I'm assuming this is the container that contains all the stuff.

Since everything else is a child (never used outside the pie chart code), the classes should be named with a `-` prefix. i.e. `& .-piechart`, but there's more.

A `.chart-pie` likely only contains a single chart, so `& .-chart` is an acceptable class name for the child.

Finally, it's likely `& .-chart` is always a direct descendant of `.chart-pie`, so it should include a `>`.

i.e. `& > .-chart {`